app/src/main/java/com/github/se/project/ui/profile/CreateUserProfile.kt

```diff
@@ -261,18 +255,26 @@ fun CreateProfileScreen(
               }
 
               if (phoneNumber.isNotEmpty() && isPhoneNumberValid(phoneNumber)) {
+                errorMessage = "Please complete all the fields before creating your account!"
                 Toast.makeText(
                         context,
                         "Please complete all the fields before creating your account!",
                         Toast.LENGTH_SHORT)
                     .show()
               } else {
+                errorMessage = "Please enter a valid phone number!"
                 Toast.makeText(context, "Please enter a valid phone number!", Toast.LENGTH_SHORT)
                     .show()
               }
             }) {
               Text("Confirm your details")
             }
+        errorMessage?.let {
+          Text(
+              text = it,
+              modifier = Modifier.padding(8.dp).testTag("errorText") // Add this tag for testing
+              )
```

Comment:
I didn't find a way to test the Toast message and get coverage by doing it. By assigning a text variable I am able to test every Toast Error message and therefore improve coverage.

Reply:
Yes, I see your issue but displaying a text in the middle of the screen seems very weird for the user experience. I think that if you juste check that the navigation has not occured it will implictly test if some exceptions are thrown and so we can remove the Text composable and keep only the toast. I will update this as well.